File: generators/service-personalityInsights/templates/node/instrumentation.js
var IBMCloudEnv = require('ibm-cloud-env');
var PersonalityInsightsV3 = require('watson-developer-cloud/personality-insights/v3');

module.exports = function(app, serviceManager){
    var personalityInsights = new PersonalityInsightsV3({
	    url: IBMCloudEnv.getString('personalityInsights_url'),
	    username: IBMCloudEnv.getString('personalityInsights_username'),
        password: IBMCloudEnv.getString('personalityInsights_password'),
        version_date: '2016-10-19'
    });
    serviceManager.set("watson-personality-insights", personalityInsights);
};

File: generators/service-personalityInsights/templates/python/instrumentation.py
from ibmcloudenv import IBMCloudEnv
from watson_developer_cloud import PersonalityInsightsV3

personality_insights = PersonalityInsightsV3(
    username=IBMCloudEnv.getString('personalityInsights_username'),
    password=IBMCloudEnv.getString('personalityInsights_password'),
    version='2016-10-20')

def getService(app):
    return 'watson-personality-insights', personality_insights